applications/tari_dan_wallet_daemon/src/jrpc_server.rs

```diff
@@ -101,6 +102,11 @@ async fn handler(
             "get_all_jwt" => call_handler(context, value, token, rpc::handle_get_all_jwt).await,
             _ => Ok(value.method_not_found(&value.method)),
         },
+        Some(("settings", method)) => match method {
+            "get_indexer_url" => call_handler(context, value, token, settings::handle_get_indexer_url).await,
```

Comment:
We may want to have `"get"` and `"set"` operations for settings. Then for now we only have the indexer_url but later we can add more, rather than having get/set handlers for each setting.

Reply:
done

Reply:
Thanks